src/pages/Login/Login.jsx

```diff
@@ -1,7 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
 
 const Login = () => {
-  return <div>로그인 페이지 입니다.</div>;
+  // ID(이메일)
+  const [id, setID] = useState('');
+  const saveUserID = event => {
+    setID(event.target.value);
+  };
```

Comment:
아래처럼 구조분해할당해서 사용할 수 있습니다.
```suggestion
  const saveUserID = event => {
  const {value} = event.target
    setID(value);
  };
```